### ISSUE TITLE
Point cloud map and vector map for Autoware

### ISSUE THREAD

**Issue by peng0701** [lgsvl/simulator#17]
Hi,
Will you provide point cloud map and vector map for Autoware?
Thanks.

**Comment by andytolst** [comment]
yes, we will provide map data and instructions how to connect Autoware and Simulator.
